apps/src/templates/sectionAssessments/MatchAssessmentsOverviewTable.jsx

```diff
@@ -101,7 +101,11 @@ class MatchAssessmentsOverviewTable extends Component {
   };
 
   optionFormatter = (option, {rowData, columnIndex, rowIndex, property}) => {
-    return <div>{`${option}`}</div>;
+    if (option.includes('<img')) {
+      return <div>{$(option).attr('alt')}</div>;
```

Comment:
Do the images always have alt text?

Reply:
No - in fact we don't have any questions of this type currently but this was the solution in the spec. I was thinking about going in and if you are writing a match question with images that we force you to have an alt.